Neos.ContentRepository.Core/Classes/Feature/RebaseableCommands.php

```diff
@@ -67,11 +72,37 @@ public function separateMatchingAndRemainingCommands(
     }
 
     private static function commandMatchesAtLeastOneNode(
-        MatchableWithNodeIdToPublishOrDiscardInterface $command,
-        NodeIdsToPublishOrDiscard $nodeIds,
+        RebasableToOtherWorkspaceInterface $command,
+        NodeAggregateIds $nodeIdsToMatch,
     ): bool {
-        foreach ($nodeIds as $nodeId) {
-            if ($command->matchesNodeId($nodeId)) {
+        foreach ($nodeIdsToMatch as $nodeId) {
+            /**
+             * This match must contain all commands which are working with individual nodes, such that they are
+             * filterable whether they are applying their action to a NodeIdToPublish.
```

Comment:
I don't get what this means. What does "whether they are applying their action to a NodeIdToPublish" mean? AFAIS we have no "NodeIdToPublish" anymore.